jni-gen/src/generators/method.rs

```diff
@@ -190,9 +190,12 @@ fn generate(
         method_name,
         class.full_name()
     ));
-    code.push("///".to_string());
-    code.push("/// Type and Java signature of parameters:".to_string());
-    code.push("///".to_string());
+
+    if !parameter_names.is_empty() {
+        code.push("///".to_string());
+        code.push("/// Type and Java signature of parameters:".to_string());
+        code.push("///".to_string());
+    }
```

Comment:
When a method does not have any parameters it will not print the `/// Type and Java signature of parameters` followed by no parameters.